index.bs

```diff
@@ -64,13 +64,6 @@ spec: TokenBinding; urlPrefix: https://tools.ietf.org/html/draft-ietf-tokbind-pr
         text: Token Binding
         text: Token Binding ID; url: section-3.2
 
-spec: WebCryptoAPI; urlPrefix: https://www.w3.org/TR/WebCryptoAPI/
-    type: dfn
-        text: normalizing an algorithm; url: dfn-normalize-an-algorithm
-        text: recognized algorithm name
-    type: dictionary
-        text: AlgorithmIdentifier; url: dfn-AlgorithmIdentifier
-
```

Comment:
Where we are using WebCrypto "recognized algorithm names" for the hash algorithms (which are listed at http://www.w3.org/TR/WebCryptoAPI/#sha-registration - such as "SHA-256"), maybe we should give an example value, such as "SHA-256".

Interestingly, the Travis-CI build succeeded for the PR without Bikeshed errors.